product_variant_change_attribute_value/wizards/product_variant_attribute_value_action.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class ProductVariantAttributeValueAction(models.TransientModel):
+    _name = "variant.attribute.value.action"
+    _description = "Wizard action to do on variant attribute value"
+
+    product_attribute_value_id = fields.Many2one("product.attribute.value",)
+    attribute_action = fields.Selection(
+        [("delete", "Delete"), ("replace", "Replace"), ("do_nothing", "Do Nothing")],
+        default="do_nothing",
+        required=True,
+    )
+    attribute_id = fields.Many2one(
+        "product.attribute",
+        related="product_attribute_value_id.attribute_id",
+        readonly=True,
+    )
+    selectable_attribute_value_ids = fields.Many2many(
+        "product.attribute.value", compute="_compute_selectable_attribute_value_ids"
+    )
+    replaced_by = fields.Many2one(
+        "product.attribute.value",
+        string="Replace with",
+        domain="[('id', 'in', selectable_attribute_value_ids)]",
+    )
+
+    @api.depends("attribute_action")
+    def _compute_selectable_attribute_value_ids(self):
+        for rec in self:
+            attribute_ids = self.attribute_id.value_ids.ids
```

Comment:
1. there's nothing depending on `attribute_action`
2. I think we can drop `compute` and make this field related to `attribute_id`